### PR TITLE
Stable to develop

### PR DESCRIPTION
**Pull request by ReimarBauer**
updating meta.yaml and other fixes

### CODE REVIEW

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -31,16 +31,16 @@ jobs:
         && conda config --add channels defaults \
         && conda install --revision 0 \
         && conda install mamba \
-        && mamba install --file reqs.txt \
+        && conda install --file reqs.txt --force-reinstall \
```
Comment (ReimarBauer) [inline .github/workflows/testing.yml]: mamba crashed with force-reinstall

File: .github/workflows/testing.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-        
+
     - name: Reinstall dependencies if changed
       run: |
         cmp -s /meta.yaml localbuild/meta.yaml && cmp -s /development.txt requirements.d/development.txt || (echo Dependencies differ \
```
Comment (ReimarBauer) [inline .github/workflows/testing.yml:23]: this seems to be the origin meta.yaml not the new one

File: .github/workflows/testing.yml (hunk 1 above)
Comment (ReimarBauer) [reply .github/workflows/testing.yml]: and at all force-reinstall is not needed, because of conda install --revision 0

**Comment by ReimarBauer** [comment]
reproduced with the docker image. 
now lets see if the reason is the order change

**Comment by ReimarBauer** [comment]
testing changes need afterwards a backport to stable

**Review by joernu76 — APPROVED** [review]

**Pull request merged**